include/xten/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@
 
 add_subdirectory(Dialect)
 add_subdirectory(Conversion)
-add_subdirectory(Transform)
\ No newline at end of file
+add_subdirectory(Transform)
```

Comment:
For reasons of consistency, this folder is usually called `Transforms`.

I also want to note my use of the `Interfaces` folder, which is not standard, but hadn't been questioned yet. Interfaces aren't subject to dialect filtering and thus harder to re-use across dialects in TableGen - they can't be mixed with other stuff. That is why I prefer to keep them separate, as they should compile without any dialect-specific TableGen includes (maybe enums at most).

Reply:
I also like to approach to keep them seperated